hswidget/cofwidget/helpers.go

```diff
@@ -0,0 +1,58 @@
+package cofwidget
```

Comment:
IMO all this file should be placed in hsutil

Reply:
the speed and duration funcs should not, and i've moved the imagebutton hack into hsutils, this is resolved.